src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/utils.ts

```diff
@@ -64,3 +50,27 @@ export const isExpenditureFullyFunded = (expenditure?: Expenditure | null) => {
     return amount.lte(tokenBalance?.amount ?? 0);
   });
 };
+
+// @todo: update steps
```

Comment:
Can this comment be expanded to include details on what needs to be updated?

Reply:
I removed this comment because all the cases we are currently working on are covered.